app/src/main/java/com/mathroule/demoapp/MainActivity.java

```
@@ -28,6 +31,26 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            setButonLable(text);
+
+            if (text == "something") {
+                setButonLable("something");
+            }
+        } catch (Exception e) {
```

<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/15d3b9565f791bb5c2bdb22540e7aa2ee8420984/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L42"></span>Empty catch block.</td>
    </tr>
  </tbody>
</table>